proposals/stage-2/CODE_OF_CONDUCT/HANDLING-REPORTS.md

```diff
@@ -29,52 +34,40 @@ Any member of the CPC or Code of Conduct Panel(CoCP) who is involved in the repo
-     3. Information is discussed: The facts are discussed in context of opinions. This can be done in the issue for the report, or
-        in a meeting in which case the key discussion points should then be added to the issue.
+     3. Information is discussed: The facts are discussed in context of opinions. This can be done in the issue for the report, or in a meeting in which case the key discussion points should then be added to the issue.
      4. An action to be taken is arrived at: The action to be taken is decided by consensus as per the standard CPC
         [Decision Making](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#section-9-decision-making) process substituting in CoCP for references to the CPC.
      5. The proposed action is discussed with the leadership for the space to which the report applies (ex CPC or member
-        project leadership). If the leadership representatives expresses concerns then the
-        contact person will bring these back to the CoCP members for further discussion.  This cycle can continue  until the 
-        CoCP members reach consensus that the leadership representatives concerns have been adequately addressed.
-     6. Resolution offered: The resolution is offered to the reportee by the contact person. If the resolution is accepted as
-        reasonable or there is no response within 7 days the process moves to the next step.  If the reportee expresses concerns then the
-        contact person will bring these back to the CoCP members for further discussion. This cycle can continue until the 
-        CoCP members reach consensus that the reportee's concerns have been adequately addressed. This may also require that
-        step 5 be repeated as well if the proposed action is modified.
-     6. Final resolution: the resolution agreed by the CoCP members is implemented and the outcome reported back to the reportee by
-        the main contact.
+        project leadership). If the leadership representatives expresses concerns then the contact person will bring these back to the CoCP members for further discussion.  This cycle can continue until the CoCP members reach consensus that the leadership representatives concerns have been adequately addressed.
+     6. Resolution shared: The resolution is shared with the reporter or target by the contact person. If the reporter or target expresses concerns then the contact person will bring these back to the CPC members for further discussion. This cycle can continue until the CPC members reach consensus that the reporter or target's concerns have been adquately addressed. If no feedback is received within 7 days the resolution is considered as accepted. 
+     6. Final resolution: the resolution agreed by the CoCP members is implemented and the outcome reported back to the reporter or target bythe main contact.
```

Comment:
```suggestion
     7. Final resolution: the resolution agreed by the CoCP members is implemented and the outcome reported back to the reporter or target bythe main contact.
```

Or we could just number everything `1.` and let markdown renderers take care of the numbering.